src/iperf_error.c

```diff
@@ -350,6 +350,10 @@ iperf_strerror(int int_errno)
             snprintf(errstr, len, "send timeout value is incorrect or not in range");
             perr = 1;
             break;
+        case IEUDPFILETRANSFER:
+            snprintf(errstr, len, "cannot transfer file using UDP");
+            perr = 1;
```

Comment:
Don't want to set `perr = 1` here, there's no reason to output the result of perror(3) because no library calls were made to cause this error (it's the user giving a bad combination of command-line arguments).

Reply:
Done.  Removed the `perr = 1`.

Reply:
Great, thanks...I think we're good to merge at this point.